rosdep/base.yaml

```diff
@@ -1811,6 +1811,16 @@ libhdf5-dev:
   debian: [libhdf5-dev]
   fedora: [hdf5-devel]
   ubuntu: [libhdf5-dev]
+libi2c-dev:
+  arch:
+    pacman:
+      packages: [linux-api-headers]
+  debian:
+    apt:
+      packages: [libi2c-dev]
+  ubuntu:
+    apt:
+      packages: [libi2c-dev]
```

Comment:
We prefer the most compact version since it usually requires the least maintenance and is the most readable. And it's not ambiguous. Every platform has a default package manager.